src/bundle/Resources/public/scss/ibexa.scss

```diff
@@ -130,3 +130,5 @@
 @import 'global-search';
 @import 'user-name';
 @import 'user-profile';
+@import 'user-mode-badge';
```

Comment:
Isn't that resource missing? Currently, `yarn encore dev` fails while tryin' to load this one.